file-browser/package.json

```diff
@@ -22,17 +22,19 @@
   },
   "private": true,
   "dependencies": {
+    "electron-is-dev": "^0.2.0",
```

Comment:
I got this one. See comment below, the config session might need to be reviewed.